### ISSUE TITLE
Configurable Font Face

### ISSUE THREAD

**Issue by Stoppablemurph** [yakyak/yakyak#980]
I don't mind the font that YakYak uses, but it would be really nice to be able to explicitly choose any desired font that's installed on the system for conversations and the conversations sidebar.

**Comment by nedclimaterisk** [comment]
Yeah, this is very important on HiDPI screens - Yakyak is quite hard to read. I could use KDE's scaling, but that screws with other toolkits.

**Comment by nedclimaterisk** [comment]
Bah, never mind me, I just found the option in `View > Font Size`. Sorry for the noise.
